master.yaml

```diff
@@ -36,13 +27,36 @@ coreos:
     - name: rpc-statd.service
       enable: true
       command: start
+    - name: etcd3.service
+      command: start
+      content: |
+        [Unit]
+        Description=etcd
+        Documentation=https://github.com/coreos/etcd
+        [Service]
+        Environment='ETCD_IMAGE_TAG=latest'
```

Comment:
This should be the version officially supported by Kubernetes. Going with `latest` may mean in the feature that we run a new, incompatible version, e.g. `4.x`.